tests/test_global_config.py

```diff
@@ -50,32 +51,32 @@
 
 
 @pytest.fixture
-def config_instance():
+def config_instance(mocker):
     @Grabber.grab_yaml_config
     def config_parser():
         return yaml_config
 
     @Grabber.grab_cli_config
     def cli_parser(
+        click_context: click.Context,
         test_configuration_file,
         log_path,
         log_level,
         report_type,
-        variant,
-        branch_level,
         pattern,
     ):
         return None
 
     config_parser()
+    context_mock = mocker.patch("click.Context")
+    context_mock.args = ["--variant", "variant1,", "--branch-level", "daily"]
```

Comment:
Where are the assertions? 🤔

Reply:
I just changed the implementation of the fixture to replace the fact that the tag and branch level were put as input parameters for the function which is not the case for pykiso CLI, so the test will be runned with the same logic as the cli.
But I added a new test.